webpack.config.js

```diff
@@ -3,7 +3,7 @@ const path = require("path");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 const ExtractTextPlugin = require("extract-text-webpack-plugin");
 
-module.exports = {
+const widgetConfig = {
     entry: "./src/PullToRefresh/widget/PullToRefresh.ts",
     output: {
         path: path.resolve(__dirname, "dist/tmp"),
```

Comment:
in webpack 2 you don't need to resolve the extension `".json" `
and probably also no `js`